Makefile

```diff
@@ -24,16 +24,16 @@ BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
 endif
 BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 
-# IMAGE_TAG_BASE defines the docker.io namespace and part of the image name for remote images.
+# IMAGE_BASE defines the docker.io namespace and part of the image name for remote images.
 # This variable is used to construct full image tags for bundle and catalog images.
 #
 # For example, running 'make bundle-build bundle-push catalog-build catalog-push' will build and push both
 # sustainable.computing.io/kepler-operator-bundle:$VERSION and sustainable.computing.io/kepler-operator-catalog:$VERSION.
-IMAGE_TAG_BASE ?= quay.io/sustainable.computing.io/kepler-operator
+IMAGE_BASE ?= quay.io/sustainable.computing.io/kepler-operator
 
 # BUNDLE_IMG defines the image:tag used for the bundle.
 # You can use it as an arg. (E.g make bundle-build BUNDLE_IMG=<some-registry>/<project-name-bundle>:<tag>)
-BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
+BUNDLE_IMG ?= $(IMAGE_BASE)-bundle:v$(VERSION)
```

Comment:
```suggestion
BUNDLE_IMG ?= $(IMAGE_BASE)-bundle:$(VERSION)
```

Reply:
@vprashar2929 rather than I think we should go ahead with the one you did already .. which is to prepend (v)